src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+title: Polices d'écriture
+description: Chapitre Polices d'écriture du web Almanac 2019 couvrant l'endroit d'où les polices sont chargées, les formats de police, la performance de leur chargement, les polices variables et les polices de couleur.
+authors: [zachleat]
+reviewers: [hyperpress, AymenLoukil]
+analysts: [tjmonsi, rviscomi]
+editors: [bazzadp]
+translators: [arigaud-ca]
+discuss: 1761
+results: https://docs.google.com/spreadsheets/d/108g6LXdC3YVsxmX1CCwrmpZ3-DmbB8G_wwgQHX5pn6Q/
+zachleat_bio: Zach is a Web Developer with <a hreflang="en" href="https://www.filamentgroup.com/">Filament Group</a>. He’s currently fixated on <a hreflang="en" href="https://www.zachleat.com/web/fonts/">web fonts</a> and <a hreflang="en" href="https://www.zachleat.com/web/introducing-eleventy/">static site generators</a>. His <a hreflang="en" href="https://www.zachleat.com/web/speaking/">public speaking résumé</a> includes talks in eight different countries at events like JAMstack_conf, Beyond Tellerrand, Smashing Conference, CSSConf, and <a hreflang="en" href="https://www.zachleat.com/web/whitehouse/">The White House</a>. He also helps herd <a hreflang="en" href="http://nejsconf.com/">NEJS CONF</a> and the <a hreflang="en" href="http://nebraskajs.com">NebraskaJS</a> meetup.
+featured_quote: Web fonts enable beautiful and functional typography on the web. Using web fonts not only empowers design, but it democratizes a subset of design, as it allows easier access to those who might not have particularly strong design skills. However, for all the good they can do, web fonts can also do great harm to your site's performance if they are not loaded properly.
+featured_stat_1: 74.9%
+featured_stat_label_1: 3P Font Requests that use Google Fonts
+featured_stat_2: 29%
+featured_stat_label_2: Percent of pages that include a Google Fonts stylesheet link
+featured_stat_3: 718
+featured_stat_label_3: Most font requests on a single page
+---
+
+## Introduction
+
+Les polices Web permettent une typographie belle et fonctionnelle sur le web. L'utilisation des polices Web permet non seulement de renforcer le design, mais aussi de démocratiser un sous-ensemble du design, car elle permet un accès plus facile à ceux qui n'ont pas forcément de solides compétences en matière de design. Cependant, malgré tout le bien qu'elles peuvent faire, les polices web peuvent également nuire considérablement aux performances de votre site, si elles ne sont pas chargées correctement.
+
+Sont-elles un atout pour le web ? Offrent-elles plus d'avantages que d'inconvénients ? Les sentiers des standards du web sont-ils suffisamment pavés pour encourager les meilleures pratiques de chargement des polices web par défaut ? Et si ce n'est pas le cas, que faut-il changer ? Voyons si nous pouvons répondre à ces questions en examinant comment les polices de caractères sont utilisées sur le web aujourd'hui.
```

Comment:
Remarques générales : si tu peux mettre un `&nbsp;` avant `:`, `;`, `!`, `?`, `%`, etc. et si possible utiliser la belle apostrophe => `’`

Reply:
I've added the `&nbsp;` in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2114/commits/6df484a83f05c285020d15faa5a0207a42c519e7. You no longer need to add the "belle apostrophe" as we now have a library that takes care of those for you!